common/config/azure-pipelines/ci-ios.yaml

```diff
@@ -0,0 +1,93 @@
+trigger: none
+
+variables:
+  - group: iTwin.js ios and App Center variables
+
+jobs:
+  - job:
+    timeoutInMinutes: 120
+    pool:
+      vmImage: macos-latest
+    workspace:
+      clean: all
+
+    steps:
+      - task: InstallAppleProvisioningProfile@1
+        displayName: 'Install an Apple provisioning profile'
+        inputs:
+          provProfileSecureFile: '$(ProvisioningProfileSecureFile)'
+
+      - task: InstallAppleCertificate@2
+        displayName: 'Install an Apple certificate'
+        inputs:
+          certSecureFile: '$(CertificateSecureFile)'
+          certPwd: '$(CertificatePassword)'
+          setUpPartitionIdACLForPrivateKey: false
+
+      - checkout: self
+        clean: true
+
+      - task: NodeTool@0
+        displayName: "Use Node 14.17.4"
+        inputs:
+          versionSpec: 14.17.4
+          checkLatest: true
+
+      - script: |
+          git config --local user.email imodeljs-admin@users.noreply.github.com
+          git config --local user.name imodeljs-admin
+        displayName: git config
+
+      - script: node common/scripts/install-run-rush.js install
+        displayName: rush install
+
+      - script: node common/scripts/install-run-rush.js build -v -p max
```

Comment:
To make this run faster, use the `--to core-backend` to only build up to that point instead of all the projects/packages.